File: Models/Candidat.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace FodApi.Model
{
    public class Candidat
    {
        [Key]
        public int id { get; set; }
        [Column(TypeName ="nvarchar(20)")]
        [Required]
        public string photo { get; set; }
        [Column(TypeName = "nvarchar(100)")]
        public string nom { get; set; }
        [Column(TypeName = "nvarchar(100)")]
        [Required]
        public string prenom { get; set; }
        [Column(TypeName = "nvarchar(100)")]
        [Required]
        public string email { get; set; }
        [Column(TypeName = "nvarchar(100)")]
        [Required]
        public int experience { get; set; }
        public string date_de_naissance { get; set; }

    }
}

File: View/src/app/model/Candidat.ts
import {Candidature} from "../model/candidature";


export class Candidat {
    id: number;
    nom:string;
    prenom:string;
    email:string;
    cv:string;
    experience:number;
    date_de_naissance:string;
    photo:string;
    constructor(id:number,nom:string,prenom:string,email:string){
        this.nom=nom;
        this.id=id;
        this.prenom=prenom;
        this.email=email;
        
    }

    }